weave-js/src/components/PagePanelComponents/Home/Browse3/views/Leaderboard/LeaderboardGrid.tsx

```diff
@@ -540,6 +540,8 @@ export const LeaderboardGrid: React.FC<LeaderboardGridProps> = ({
     });
   }, [columns, columnOrder]);
 
+  const [sortModelInitialized, setSortModelInitialized] = useState(false);
```

Comment:
Since you don't want setting this value to trigger a render `useRef(false)` could be a better approach